### ISSUE TITLE
Files from /etc added in RUN do not get copied into snapshot

### ISSUE THREAD

**Issue by dzoeteman** [GoogleContainerTools/kaniko#457]
**Actual behavior**
Files added in RUN command to the `/etc` directory do not get added to the snapshot.

**Expected behavior**
Files in `/etc` directory added by RUN command should be added to snapshot.

**To Reproduce**
1. Use Dockerfile below for a build
2. Notice in build log that the two `ls` commands both show files in directory
3. Open shell in the built image
4. Notice that `/var/test/testfile` exists, but `/etc/test/testfile` doesn't.

**Additional Information**
 - Dockerfile 

```
FROM debian

RUN mkdir -p /var/test \
    && mkdir -p /etc/test \
    && touch /var/test/testfile \
    && touch /etc/test/testfile \
    && ls -lah /var/test \
    && ls -lah /etc/test;
```

 - Build Context

No additional files needed.

 - Kaniko Image (fully qualified with digest)

`gcr.io/kaniko-project/executor:latest` (`gcr.io/kaniko-project/executor@sha256:434bbb1d998ba1bd8ebc04c90d93afa859fd5c7ff93326bca9f6e7da0d6277ff`)

This is a pretty big issue. Any installs where config is added in `/etc` will not work properly. How I found this out for example, was that I tried to install `nginx`, but the `/etc/nginx` directory was missing from the image.
The issue is not in release `v0.6.0`, which leads me to believe as #451 might be the problem? That's a pure guess though.

**Comment by slavoutich** [comment]
I am experiencing kinda the same issue, but with a different flavour. I have a base image, where I `ADD` a file to `/etc/init.d`, where it is added successfully, and I have an image constructed `FROM` base image, where it magically disappears. I think that this should be the same bug, actually, but can try to open separate bug with more detailed info upon request.
```
Pulling docker image gcr.io/kaniko-project/executor:debug ...
Using docker image sha256:6e43810f8da1b757c553ca5d8f70956f127a6eb70f1400fca537db32ef89e7ca for gcr.io/kaniko-project/executor:debug ...
```

**Comment by dzoeteman** [comment]
Can confirm this is fixed with #461, so this can be closed.
